vault/resource_namespace.go

```diff
@@ -19,12 +19,6 @@ func namespaceResource() *schema.Resource {
 		},
 
 		Schema: map[string]*schema.Schema{
-			"id": {
```

Comment:
Hey! I'm curious why this is stripped. [There's a general process](https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-attribute-removal) that Terraform usually performs when removing an attribute that we may want to follow here depending on the situation.

I did run the `resource_namespace_test.go` file against the current version of Vault enterprise and it did pass, so that piece is looking good.

Reply:
Ah yes! `id` is a special attribute that the SDK actually automatically provides. It does not need to be declared in the schema, when upgrading from sdk v0.10 to v0.11 it is no longer allowed to declare it in the schema. So this is not a breaking change all configs can access the `id`!

Reply:
:+1: